daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_expressions/expressions.tdml

```diff
@@ -2408,8 +2408,8 @@
     <!--  This test is incorrect. An element of simple type cannot use "." 
     in any inputValueCalc, as "." means "the value of this element", so 
     we're chasing our tail here. -->
-    <xs:element name="e8" type="xs:int"
-      dfdl:inputValueCalc="{ dfdl:checkConstraints(.) }" />
+<!--    <xs:element name="e8" type="xs:int"-->
```

Comment:
This one doesn't have the problematic IVC. So the comment above "This test is incorrect" doesn't match does it?
Fix comment(moves up to before the prior element?), remove unused element decl.